lib/csv/writer.rb

```diff
@@ -31,6 +31,8 @@ def <<(row)
       @headers ||= row if @use_headers
       @lineno += 1
 
+      apply_converters_to(row, lineno)
```

Comment:
We need to receive converted result such as `row = @converter.convert(row, ...) if @converter`.